extensions/vscode/src/views/homeView.ts

```diff
@@ -1068,15 +1068,21 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
         }
 
         let details = [];
-        if (!isRelativePathRoot(contentRecord.projectDir)) {
-          details.push(`${contentRecord.projectDir}${path.sep}`);
-        }
         if (credential?.name) {
           details.push(credential.name);
         } else {
           details.push(`Missing Credential for ${contentRecord.serverUrl}`);
           problem = true;
         }
+        if (
+          !isRelativePathRoot(contentRecord.projectDir) &&
+          config &&
+          !isConfigurationError(config)
+        ) {
+          details.push(
+            `${contentRecord.projectDir}${path.sep}${config.configuration.entrypoint}`,
+          );
+        }
```

Comment:
Good catch! I've updated the logic and rather than optionally appending it, I've gone with a simple if statement. Given the two choices, it seemed the if statement was more readable.

Reply:
Looks like the update didn't include this suggestion

> If we have a valid configuration then add ${path.sep}${config.configuration.entrypoint} to the end of it

At the relative path root I'm not seeing the entrypoint in the selection at all:

![CleanShot 2024-09-20 at 12 20 27@2x](https://github.com/user-attachments/assets/42dd8019-f95d-4637-88d6-2bb1d97f1504)